backend/uclapi/dashboard/views.py

```diff
@@ -43,6 +44,8 @@ def shibboleth_callback(request):
         )
 
         new_user.save()
+        add_user_to_mailing_list(new_user.email, new_user.given_name)
```

Comment:
could you make this display name instead of given name?

Reply:
whats display name?

Reply:
oh full name. cool